pywbem/mof_compiler.py

```diff
@@ -2119,8 +2298,7 @@ def CreateClass(self, *args, **kwargs):
                               IncludeQualifiers=True)
             except CIMError as ce:
                 if ce.status_code == CIM_ERR_NOT_FOUND:
-                    ce.args = (CIM_ERR_INVALID_SUPERCLASS, cc.superclass)
-                    raise
+                    raise CIMError(CIM_ERR_INVALID_SUPERCLASS, cc.superclass)
```

Comment:
As long as we are modifying messages why not add target classname rather than just the superclassname

Reply:
I agree. This boils down to specifying a reasonable human readable status description. There are several other cases in this class that have similar issues (status description contains only a very short string without being really understandable unless you know how everything works). I suggest to improve that in a separate PR. I have created issue #2014 for that.